### DISCUSSION TITLE
20231118 VU LED not working

### DISCUSSION THREAD

**Discussion by noriadzik** [rern/rAudio#1505] [Fixed Issues]
Setup1: RPi 4 + DigiAMP+ + 20231118
Setup2: RPi 4 + 20231105

VU don't set voltage to the specified pins while music is playing. Tested with default pins 8/10/12/16/18/22/24 on setup2 and with ping 11/19/21/23/29/31/33 on setup1 (to avoid conflicts with HAT).

**Comment by noriadzik** [comment]
In 20231128 LEDs are working fine.

**Reply by viktor6** [reply]
does not work in release 20250404
